conan/tools/gnu/makedeps.py

```diff
@@ -597,6 +603,7 @@ class MakeDeps:
     """
 
     _title = "# This Makefile has been generated by Conan. DO NOT EDIT!\n"
+    _footer = f"# end {CONAN_MAKEFILE_FILENAME}\n"
```

Comment:
Not sure about the value of this footer. No other Conan generated file adds this end-of-file footer, so better remove it to align.

Reply:
I'll remove it, but I would advise adopting this as a general convention. A clear footer indicates that you have the entire file, that it was not truncated for one reason or another.

Reply:
We haven't been reported a truncated file ever in many years. Creation of generated files happen in memory, and written all at once in single operation, extremely unlikely to happen.